index.src.html

```diff
@@ -352,32 +344,18 @@ <h3 id="header">
   [[!RFC5234]]:
 
   <pre class="abnf" link-type="grammar" dfn-type="grammar">
-    Report-To = <a>json-field-value</a>
-                ; See Section 2 of [[HTTP-JFV]], and Section 2 of [[RFC7159]]
+    Clear-Site-Data = "Clear-Site-Data" ":" <a>clear-site-data-value</a>
+    <dfn>clear-site-data-value</dfn> = OWS 1#(<a>type</a> OWS)
+    <dfn>type</dfn> = atom
```

Comment:
> -> From what I have read in other specs so far, the definition of a header always includes the key and separator (i.e. "Clear-Site-Data" ":"), not just the value

I'm following along with things like https://tools.ietf.org/html/rfc7231#section-3.1.2.2 here. I'm sure I've done it differently elsewhere, but this is what I think IETF folks have solidified upon for header ABNF.

> -> The "data-type" and "extension-type" tokens do not need the suffix "-value" (they represent the datatypes, not the header value as a whole, so the name is expressive enough)

Sure.

> -> I still think we need to specify OWS; I don't think spaces around tokens should be a syntax error

This is part of the `#` rule in https://tools.ietf.org/html/rfc7230#section-7 which we should be linking to. @annevk suggested elsewhere that we link to https://fetch.spec.whatwg.org/#abnf to make that clear.

> -> You suggested that alpha is wrapped in anchors; do you want it to link to RFC5234?

Yes, please. Just copy/paste from https://github.com/w3c/webappsec-csp/blob/master/index.src.html#L101. :)